src/test/java/io/openliberty/tools/intellij/it/fixtures/ProjectFrameFixture.java

```diff
@@ -241,7 +270,20 @@ public ContainerFixture getDocumentationHintEditorPane() {
      * @return The ContainerFixture object associated with the DocumentationHintEditorPane pop-up window.
      */
     public ContainerFixture getDiagnosticPane() {
-        return find(ContainerFixture.class, byXpath("//div[@class='HeavyWeightWindow']//div[@class='JEditorPane']"), Duration.ofSeconds(20));
+        return find(ContainerFixture.class, byXpath("//div[@class='HeavyWeightWindow']//div[@class='JEditorPane']"), Duration.ofSeconds(5));
+    }
+
+    /**
+     * Returns the ContainerFixture object associated with the DocumentationHintEditorPane pop-up window.
+     *
+     * @return The ContainerFixture object associated with the DocumentationHintEditorPane pop-up window.
+     */
```

Comment:
this method description is the same as getQuickFixPane() - I'm assuming this one needs to be updated?